docs/getting-started/Legacy_ESLint_Setup.mdx

```diff
@@ -3,13 +3,16 @@ id: legacy-eslint-setup
 title: Legacy ESLint Setup
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```

Comment:
[Non-Actionable] Yeah I don't really know how this was working before...

Reply:
It was showing up as warnings in the console when I did `yarn docusaurus build` which is why I fixed it.

Idk mdx/docusaurus must do some "helpful" stuff sadly.